### PR TITLE
added ignore pgs subs setting to embedded scan

### PR DESCRIPTION
**Pull request by sekkr1**
Added feature to ignore pgs type subtitles when looking at embedded subtitles as they almost always require burn in through plex which is not ideal.

### CODE REVIEW

File: bazarr/config.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
         'page_size': '25',
         'minimum_score_movie': '70',
         'use_embedded_subs': 'True',
+        'ignore_pgs_subs': 'True',
```
Comment (morpheus65535) [inline bazarr/config.py]:
```suggestion
        'ignore_pgs_subs': 'False',
```

File: views/settings.tpl [hunk 2]
```diff
@@ -1188,6 +1188,25 @@
                             </div>
                         </div>
 
+                        <div class="middle aligned row">
+                            <div class="right aligned four wide column">
+                                <label>Ignore embedded PGS subtitles</label>
+                            </div>
+                            <div class="one wide column">
+                                <div id="settings_ignore_pgs" class="ui toggle checkbox" data-ignore={{settings.general.getboolean('ignore_pgs_subs')}}>
```
Comment (morpheus65535) [inline views/settings.tpl]:
```suggestion
                                <div id="settings_ignore_pgs" class="ui toggle checkbox" data-ignorepgs={{settings.general.getboolean('ignore_pgs_subs')}}>
```

File: views/settings.tpl [hunk 3]
```diff
@@ -2232,6 +2251,12 @@
                 $("#settings_embedded").checkbox('uncheck');
             }
 
+    if ($('#settings_ignore_pgs').data("ignore") === "True") {
```
Comment (morpheus65535) [inline views/settings.tpl]:
```suggestion
    if ($('#settings_ignore_pgs').data("ignore") === "True") {
```
```suggestion
    if ($('#settings_ignore_pgs').data("ignorepgs") === "True") {
```

**Comment by sekkr1** [comment]
Nah I put it there by mistake.

**Pull request merged**